pygmt/tests/test_basemap.py

```diff
@@ -48,9 +48,7 @@ def test_basemap_loglog():
 def test_basemap_power_axis():
     "Create a power axis basemap plot"
     fig = Figure()
-    fig.basemap(
-        R=[0, 100, 0, 5000], J="x1p0.5/-0.001", B='x1p+l"Crustal age"y500+lDepth'
-    )
+    fig.basemap(R=[0, 100, 0, 5000], J="x1p0.5/-0.001", B='x1p+l"Crustal age"')
```

Comment:
I've fixed this because the original image had "Depth" instead of "Crustal Age" as the x-axis label... See https://github.com/GenericMappingTools/pygmt/pull/352/files?short_path=edaeec0#diff-edaeec03fad61caa0eca5752b8b18d02